internal/deploy/kubectl/websphereliberty-app-operator.yaml

```diff
@@ -17,28 +16,27 @@ metadata:
     app.kubernetes.io/name: websphere-liberty-operator
   name: wlo-leader-election-role
 rules:
-  - apiGroups:
-      - ""
-      - coordination.k8s.io
-    resources:
-      - configmaps
-      - leases
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - update
-      - patch
-      - delete
-  - apiGroups:
-      - ""
-    resources:
-      - events
-    verbs:
-      - create
-      - patch
-
+- apiGroups:
```

Comment:
The indentation doesn't look right. Not entirely sure if it's an issue but could be

Reply:
From what I've seen, both indentations are valid and equivalent.